tests/test_event.py

```diff
@@ -176,6 +176,22 @@ def test_start_end_same_type() -> None:
         )
 
 
+def test_no_end_time_or_dur() -> None:
+    """Verify that events with no end time or duration will use correct defaults."""
+
+    day_event = Event(
+        summary=SUMMARY,
+        dtstart=date(2022, 9, 9)
+    )
+    assert day_event.end == date(2022, 9, 10)
```

Comment:
Mind adding tests for duration and `computed_duration` while you're here?

Reply:
Good idea! Added.